library/tinkerpop/src/main/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopGraph.java

```diff
@@ -311,6 +330,19 @@ public void addVertex(final GafferPopVertex vertex) {
                 .then(new AddElements())
                 .build());
 
+        Operation getOperation = new GetAllElements.Builder()
+        .view(new View.Builder()
+                .entities(graph.getSchema().getEntityGroups())
+                .build())
+        .build();
+
+        final Iterable<? extends GafferPopElement> result = execute(new Builder()
+                .first(getOperation)
+                .then(new GenerateObjects.Builder<GafferPopElement>()
+                        .generator(new GafferPopElementGenerator(this))
+                        .build())
+                .build());
+
```

Comment:
Are these used? It doesn't look like it based on the other change. I reckon the annotations are the only changes needed in this file.

Reply:
Must have missed this, will remove. Thank you